src/bosh_azure_cpi/lib/cloud/azure/telemetry/telemetry_manager.rb

```diff
@@ -10,7 +10,7 @@ class TelemetryManager
 
     def initialize(azure_config)
       @azure_config = azure_config
-      @logger = Bosh::Cpi::Logger.new(CPI_TELEMETRY_LOG_FILE)
+      Bosh::AzureCloud::CPILogger.instance.logger = Bosh::Cpi::Logger.new(CPI_TELEMETRY_LOG_FILE)
```

Comment:
the logger is changed.. I really doubt if the logs are written to bosh director (STDERR).

Reply:
good catch.